docs/src/prototypes/MessageReactionsWithPopup/index.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Avatar, Chat } from '@stardust-ui/react'
+import ReactionPopup from './ReactionPopup'
+
+const reactions = [{ icon: 'thumbs up', content: '1K' }, { icon: 'thumbs down', content: 2 }]
+const reactionsWithPopup = _.map(reactions, reaction => render =>
+  render(reaction, (Component, props) => <ReactionPopup Component={Component} props={props} />),
+)
```

Comment:
Do we need `reactionsWithPopup()`? Can we just use `{ as: ReactionPopup, icon: 'thumbs up', content: '1K' }`?